CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+# Keen IO .NET SDK
+
+First off, thank you for considering contributing to this project. It's people like you that make Keen IO such a great tool.
+
+We put these guidelines together to try and make working with our SDK as straightforward as possible, and hopefully help you understand how we communicate about potential changes and improvements.
+
+Improving documentation, bug triaging, building modules for various frameworks or writing tutorials are all examples of helpful contributions we really appreciate.
+
+Please, don't use the issue tracker for support questions. If you have a support question please come hang out in http://keen.chat or send an email to team@keen.io
+
+## Guidelines
+
+* Create issues for any major changes and enhancements that you wish to make, after checking to make sure they don't already exist. Discuss things transparently and get community feedback.
+* Be welcoming to newcomers and encourage diverse new contributors from all backgrounds. 
+* Please take time to read our [Community Code of Conduct](https://github.com/keenlabs/keen-sdk-net/blob/master/CODE_OF_CONDUCT.md) which includes Reporting Guidelines.
```

Comment:
I think you can just use a relative link like (CODE_OF_CONDUCT.md) to link since it is in the same folder.